src/client/classroom/session/chatbox/MessageCompose.jsx

```diff
@@ -1,34 +1,71 @@
 import React from 'react';
 import { InputGroup, FormControl, Button } from 'react-bootstrap';
 import { useStates, bindState } from '../../../hooks/useStates.js';
+import { useRealtime } from '../../../contexts/RealtimeProvider.jsx';
+import { useDataStore } from '../../../contexts/DataStoreProvider.jsx';
 
 // TODO: reply to question
 
 export default function MessageCompose({ onCreateQuiz }) {
-  const data = useStates({
-    message: ''
+  const { sendMessage, sendQuestionMessage } = useRealtime();
+  const { data } = useDataStore();
+
+  const messageData = useStates({
+    message: '',
   });
 
   const onSend = () => {
-    console.log(data.message);
+    console.log('The content of message: '.concat(messageData.message));
+    sendMessage(messageData.message);
```

Comment:
Lol remember to clear text box after sending, same thing in other `send` functions
```suggestion
    sendMessage(messageData.message);
    messageData.message = '';
```